src/oaklib/implementations/semsimian/semsimian_implementation.py

```diff
@@ -56,8 +59,40 @@ def __post_init__(self):
             mn = m if isinstance(m, str) else m.__name__
             setattr(SemSimianImplementation, mn, methods[mn])
 
-        spo = [r for r in self.wrapped_adapter.relationships(include_entailed=True)]
-        self.semsimian = Semsimian(spo)
+        self.term_pairwise_similarity_attributes = [
+            attr
+            for attr in vars(TermPairwiseSimilarity)
+            if not any(attr.startswith(s) for s in ["class_", "_"])
+        ]
+
+    def _ensure_semsimian_object_is_set(
```

Comment:
The name doesn't match the logic. What happens if you call this a 2nd time with the same predicates?